src/frontend/screens/Settings/components/OtherSettings/index.tsx

```diff
@@ -159,6 +159,15 @@ export default function OtherSettings({
     </InfoBox>
   )
 
+  const steamPathInfo = (
+    <InfoBox text="infobox.help">
+      {t(
+        'help.steam_path.info',
+        'This path lets Heroic determine what version of Proton Steam uses, for adding non-Steam games to Steam.'
```

Comment:
I feel replacing the comma with an "and" will make it more understandable since these two are separate things.

Reply:
How about:

Heroic uses this path to determine what version of Proton Steam uses, and, to add non-Steam games to Steam.

Reply:
Looks good.